src/score/score.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Inject, Injectable } from '@nestjs/common';
-import { ClientKafka, MessagePattern, Payload } from '@nestjs/microservices';
+import { ClientKafka, MessagePattern, EventPattern, Payload } from '@nestjs/microservices';
```

Comment:
Why is EventPattern being imported here?

Reply:
Sorry, gonna remove it.